src/index.js

```diff
@@ -103,8 +107,6 @@ class ReactNativeModal extends Component {
   state = {
     showContent: true,
     isVisible: false,
-    deviceWidth: Dimensions.get("window").width,
-    deviceHeight: Dimensions.get("window").height,
```

Comment:
I think we should keep them here as well, since we need them to handle the device rotation.